cmd/upgrade.go

```diff
@@ -32,7 +32,7 @@ We were not able to upgrade the cli because we encountered an error:
 
 Please check the FAQ for solutions to common upgrading issues.
 
-https://exercism.io/faqs`, err)
+https://exercism.org/docs/using/faqs`, err)
```

Comment:
```suggestion
https://exercism.org/faqs`, err)
```

I'd rather `faqs` is the permalink and we do the redirect on the website. But the domain should still be updated :)

Reply:
Ok, but me. But as the site https://exercism.org/faqs still gave 404 I didn't thought I should like to that.